ctf-javahackfest-2015/begin/README.md

```diff
@@ -0,0 +1,6 @@
+Point: 50
+
+Pada soal pertama ini, panitia hanya memberikan sebuah file zip dan dalam file
+tersebut terdapat flag "StartJavaHackFest2015"
```

Comment:
also better make the challenges in English, I think @abdilahrf can help you with these :smile: